docs/contentful.rst

```diff
@@ -379,6 +385,71 @@ It's not problematic to have them created by non developers, it's just that they
+* ``SWITCH_CONTENTFUL_HOMEPAGE_DE`` should be set to ``True`` if you are working on the German Contentful-powered homepage
+* ``CONTENTFUL_ENVIRONMENT`` Contentful has 'branches' which it calls environments. `master` is what we use in production, and `sandbox` is generally what we use in development. It's also possible to refference a specific environment - eg ``CONTENTFUL_ENVIRONMENT=sandbox-2021-11-02``
+
+To get values for these vars, please check with someone on the backend team.
+
+If you are working on the Contentful Sync backed by the message-queue (and if you don't know what this is, you don't need it for local dev), you will also need to set the following env vars:
+
+``CONTENTFUL_NOTIFICATION_QUEUE_URL``
+``CONTENTFUL_NOTIFICATION_QUEUE_REGION``
+``CONTENTFUL_NOTIFICATION_QUEUE_ACCESS_KEY_ID``
+``CONTENTFUL_NOTIFICATION_QUEUE_SECRET_ACCESS_KEY``
+
+
+How to preview your changes on localhost
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When viewing a page in Contentful, it's possible to trigger a preview of the draft page. This is typically rendered on www-dev.allizom.org. However, that's only useful for code that's already in ``main``.
+If you want to preview Contentful content on your local machine - eg you're working on a feature branch that isn't ready for merging - do the following:
+
+* In the right-hand sidebar of the editor page in Contentful...
+* ...find the Preview section...
+* ...select ``Change`` and pick ``Localhost Preview``
+* Click ``Open preview`` to trigger a preview on your local machine.
+
+Note that previewing a page will require it to be pulled from Contentful's API, so you will need ``CONTENTFUL_SPACE_ID`` and ``CONTENTFUL_SPACE_KEY`` set in your ``.env``. It may take a few seconds to get the data.
+
+Also note that when you select ``Localhost preview``, the choice sticks, so you should set it back to ``Preview on web`` when you're done.
+
+
+How to update/refresh the sandbox environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It helps to think of Contentful 'environments' as simply branches of a git-like repo full of content. You can take a particular environment and branch off it to make a new environment for WIP or experimental content, using the original one as your starting point.
+On top of this, Contentful has the concept of aliases for environments and we use two aliases in our setup:
+
+* ``master`` is used for production and is an alias currently pointing to the `V1` environment. It is pretty stable and access to it is limited.
+* ``sandbox`` is used for development and more team members have access to edit content. Again, it's an alias and is pointed at an environment (think, branch) with a name in the format ``sandbox-YYYY-MM-DD``.
+
+
+While updating ``master`` is something that we generally don't do (at the moment only Dan B would do this), updating the sandbox happens more often, typically to populate it with data more recently added to master.
```

Comment:
Instead of a person's name can we say something like "Contentful admin" or similar?